src/parser/name_resolution.c

```diff
@@ -7059,26 +7091,21 @@ pt_resolve_using_index (PARSER_CONTEXT * parser, PT_NODE * index, PT_NODE * from
 		      PT_INTERNAL_ERROR (parser, "resolution");
```

Comment:
Shouldn't the error also be printed as pt_write_warning?

Reply:
The nature of the error is judged to be different.
Changed only for MSGCAT_SET_PARSER_SEMANTIC of the existing code.